pom.xml

```diff
@@ -352,6 +353,12 @@
                 <scope>test</scope>
             </dependency>
 
+            <dependency>
+                <groupId>com.ibm.db2</groupId>
+                <artifactId>jcc</artifactId>
+                <version>${db2.version}</version>
+            </dependency>
```

Comment:
Some Notices & License need to be added. https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/release-docs/NOTICE

Reply:
This scope need to be test, Like DM. And no need to add a license.
Is it right? @ashulin

Reply:
@CalvinKirs

Reply:
These are two different things, for new connectors we don't need to publish binary packages, so we don't have to add these, for Jdbc we don't provide this dependency, so you need to change this dependency scope to `provide`. The meaning of `test` is too vague.